### PR TITLE
fix: handle an issue where namespace casting is Capital case instead …

### PR DESCRIPTION
**Pull request by mfauzaan**
## Summary
This PR addresses an issue introduced in our previous [PR](https://github.com/mindvalley/wukong-cli/pull/217/files) due to the forced migration to `to_string_trait_impl` with the new Rust version. During this migration, all display calls were replaced with `write!(f, "prod")`. However, an error was made by mistakenly using an titlecase instead of the lowercase.

This update corrects the casing error to ensure the accurate representation of the string in all relevant locations.

## What's Changed

- [x] Corrected the string "Prod" to "prod" in all instances of write!(f, "Prod") to write!(f, "prod")
- [x] Corrected the string "Staging" to "staging" in all instances of write!(f, "Prod") to write!(f, "prod").
- [x] Corrected the string "Green" to "green" in all instances of write!(f, "Green") to write!(f, "green").
- [x] Corrected the string "Blue" to "blue" in all instances of write!(f, "Blue") to write!(f, "blue").

Ticket: -

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/221#" title="Author self-approved">mfauzaan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by Fadhil — APPROVED** [review]
Works great (Still can't get the actual list of applications, but I expect that's related to the Erlang/Elixir versioning problem) 

Looks good otherwise.

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/221#pullrequestreview-2170904666" title="Approved">Fadhil</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/221#" title="Author self-approved">mfauzaan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
